modules/react/button/stories/button/examples/CustomStyles.tsx

```diff
@@ -20,7 +20,9 @@ const getDropdownColors = () => {
       label: colors.frenchVanilla100,
     },
     active: {},
-    focus: {},
+    focus: {
+      boxShadowOuter: colors.berrySmoothie400,
```

Comment:
i would add boxShadadow inner as well just to showcase

Reply:
Agreed